tests/selectedtests/task_mappings/test_task_mappings_cli.py

```diff
@@ -19,15 +19,27 @@ def m_ns(relative_name):
 
 
 class TestCli:
+    @patch(m_ns("init_repo"))
     @patch(ns("RetryingEvergreenApi"))
     @patch(m_ns("_get_filtered_files"))
     def test_integration(
```

Comment:
I think this test would work better in the `test_task_mappings.py` test now, since it is really testing `create_mappings`.

Reply:
This is a little bit outside the intended scope of this issue but it's rather small. It's been moved